### PR TITLE
makes search work with filters

### PR DESCRIPTION
**Pull request by Danwhy**
ref #374,
Adds search query to url as part of the query string, and makes filters and search work together

### CODE REVIEW

File: web/controllers/homepage_controller.ex [hunk 1]
```diff
@@ -74,26 +86,6 @@ defmodule App.HomepageController do
...
-      |> Map.new
-      |> Map.merge(%{"subscription" => "true"})
-  end
-
-  @doc"""
-    iex>audience_map = %{"audience" => audience_params()}
-    iex>content_map = %{"content" => content_params()}
-    iex>create_query_string(Map.merge(audience_map, content_map))
-    "audience=&content=subscription"
-  """
```
Comment (shouston3) [inline web/controllers/homepage_controller.ex]: Nice move, I wasn't too much a fan of this section being in a `doctest`

**Review by samhstn — APPROVED** [review]
Looks great

**Review by nelsonic — APPROVED** [review]
@Danwhy great work! 🎉

File: web/controllers/search.ex [hunk 2]
```diff
@@ -2,7 +2,7 @@ defmodule App.Search do
   @moduledoc """
     Functions for searching through resources with a query and checking if query
     terms are in resource anywhere. Excludes stop words ('a', 'and', 'the' etc.)
-    and takes typos into account. 
+    and takes typos into account.
   """
   @stop_words "stop_words.txt" |> File.read! |> String.split(",")
```
Comment (SimonLab) [inline web/controllers/search.ex]: :+1:

**Review by SimonLab — APPROVED** [review]
LGTM :+1:

**Pull request merged**